pkg/secret_provider/managed_secret_provider.go

```diff
@@ -83,15 +73,13 @@ func (msp *ManagedSecretProvider) GetDefaultIAMToken(freshTokenRequired bool) (s
 	defer cancel()
 	defer conn.Close()
 
-	response, err := c.GetDefaultIAMToken(ctx, &sp.Request{IsFreshTokenRequired: true})
+	response, err := c.GetDefaultIAMToken(ctx, &sp.Request{IsFreshTokenRequired: freshTokenRequired})
```

Comment:
Can we change the name of `freshTokenRequired` var may be `isFreshTokenRequired`